README.md

```diff
@@ -76,37 +75,86 @@ let dog = new Dog("Spot");
 that `Dog` is an instance of `Pet`. When `super` is used as an `object` that refers to
-a `Pet` instance, it can calls methods of the parent `class` `Pet` explicitly.
+a `Pet` instance, it can call methods of the parent `class` `Pet` explicitly.
 
 ## Explain When `class` Inheritance Should Be Used in JavaScript
 
-JavaScript can be a bit confusing since it is a dynamic language that uses syntactical
+You probably won't use inheritance often, but as your code bases get larger, you are more
+likely to find a need for it. If you're creating a number of objects that have similar
+features, then creating a generic object type to contain all the shared functionality,
+and inheriting those features in more specialized objects can be convenient and useful.
+
+JavaScript can be a bit confusing since it is a dynamic language that uses syntactic
 sugar to leverage `class` implementations and inheritance similar to classic programming
 languages. However, JavaScript's inheritance model is quite powerful and fairly trivial
 to build on top of like when using a classic programming language model. As the popular
 phrase goes: "With power comes great responsibility." It can be tempting to overuse these
-features. When creating "child" `class`es, we defer to _subtyping_ relation principles
-called _(strong) behavioral subtyping_ in programming language theory. 
+features.
+
+When creating child `class`es, we defer to _subtyping_ relation principles
+called _(strong) behavioral subtyping_ in programming language theory. A parent-child 
+`class` relation is _semantic_ rather than merely _syntactic_. Let's use `Vehicle` as
+a parent `class` with `Car` as a child `class` of `Vehicle` as an example. 
+
+In _classical_ inheritance, it creates a _copy_ of the behavior from parent. However, in
+JavaScript when we create the object it does not copy the properties or behavior, it creates
+a _link_. This is called _prototypal_ inheritance. Copies of `class`es become separate entities,
+no longer directly associated with the parent.
+
+> Every object in JavaScript has an internal property called `prototype`. `prototype`s are what
+allows JavaScript objects to inherit features from one another. This is often referred to as a
+_prototype chain_. Methods and properties are not copied from one object to another in the
+prototype chain--they are accessed by _walking up_ the chain.
+
+While on the surface may look like it does the same thing, `class`es that are _linked_ and *not*
+_copied_ are fundamentally a code reuse mechanism--a way for objects to share code. The way that
+the code is shared matters. `Car`looks to the parent as a reference, and does not
+have all `Vehicle`'s properties and behaviors without it. When you attempt to access a property
+or method of an object, JavaScript will continue searching through the object/`class` and its
+`prototype` until the end of the `prototype` chain is reached. It can create performance and
+code maintenance issues when misapplied.
```

Comment:
I think this paragraph is definitely better, but still could use some clarity right around 'The way that
the code is shared matters.'

Reply:
I moved where this sentence is stated. It might make sense later on in the paragraph where I talk about the `prototype` chain.